includes/redis-connect-helper.php

```
@@ -32,159 +32,228 @@
+function wppo_redis_connect_sentinel( $config ) {
+	if ( ! class_exists( 'RedisSentinel' ) ) {
+		return new \WP_Error( 'missing_sentinel', __( 'RedisSentinel class not found.', 'performance-optimisation' ) );
+	}
+
+	$nodes = wppo_parse_nodes( $config['nodes'] ?? array() );
+	if ( empty( $nodes ) ) {
+		return new \WP_Error( 'low_nodes', __( 'Not enough Sentinel nodes configured.', 'performance-optimisation' ) );
+	}
+
+	if ( version_compare( phpversion( 'redis' ), '6.0.0', '<' ) ) {
+		return new \WP_Error( 'redis_version', __( 'Sentinel mode requires phpredis version 6.0.0 or higher.', 'performance-optimisation' ) );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '112', column '14'). (undefined)

(MissingImport)